examples/lambda/hello_lambda.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+def lambda_handler(event, context):
+    # This will show up in CloudWatch
```

Comment:
I think we should either update this message to `This will log to a CloudWatch Logs group if allowed by the Lambda role` or add the correct permission to the IAM role, as we are not defining the CloudWatch:Logs action to actually log.

In both cases, we should rewrite it to `This will log to a CloudWatch Logs group` to better explain what's going on.

What do you think?

Reply:
I will remove logging because it's not that clear cut due to the AWS console. If you run the lambda you'll see the following under **__Log output__**:

    START RequestId: dd6b2bf4-05e8-11e8-8679-a50298ecb88b Version: $LATEST
    Value of 'foo': bar
    END RequestId: dd6b2bf4-05e8-11e8-8679-a50298ecb88b
    REPORT RequestId: dd6b2bf4-05e8-11e8-8679-a50298ecb88b	Duration: 1.13 ms	Billed Duration: 100 ms 	Memory Size: 128 MB	Max Memory Used: 21 MB	

If we were to print `This will log to a CloudWatch Logs group if allowed by the Lambda role` and it appears in the console then it wouldn't be unreasonable to believe that the correct permissions are set. So I'll simplify the example.

Reply:
@oscr My question was meant for the comment line only, not the log line that would appear in CloudWatch Logs 😄, sorry for the confusion!